WordPress/Info.plist

```diff
@@ -550,8 +550,6 @@
 		<string>Nunito-Bold.ttf</string>
 		<string>EBGaramond-Regular.ttf</string>
 	</array>
-	<key>UIApplicationShortcutWidget</key>
-	<string>org.wordpress.WordPressTodayWidget</string>
```

Comment:
As far as I could gather from [the docs](https://developer.apple.com/documentation/bundleresources/information_property_list/uiapplicationshortcutwidget) and the latest [support page](https://support.apple.com/en-us/HT207122), this key was used at the time of the iOS 10 to 13 widgets but is no longer needed.